lnwallet/chainfee/estimator.go

```diff
@@ -35,6 +35,14 @@ const (
 	// maxFeeUpdateTimeout represents the maximum interval in which a
 	// WebAPIEstimator will request fresh fees from its API.
 	maxFeeUpdateTimeout = 20 * time.Minute
+
+	// WebAPIConnectionTimeout specifies the timeout value for connecting
+	// to the api source.
+	WebAPIConnectionTimeout = 5 * time.Second
+
+	// WebAPIResponseTimeout specifies the timeout value for receiving a
+	// fee response from the api source.
+	WebAPIResponseTimeout = 5 * time.Second
```

Comment:
Is the change from 10 seconds to 5 seconds here on purpose? Looks to me like this would change the behavior for Neutrino nodes (not just during the itest).

Reply:
def not on purpose🤦🏻 changed back now